app/views/works/_work_form_associations_language.html.erb

```diff
@@ -1,11 +1,12 @@
 <% # expects local variable f %>
-<dt class="language">
-  <%= f.label :language_id, ts('Choose a language') %>
+<dt class="language required">
+  <%= f.label :language_id, ts("Choose a language") %>
```

Comment:
I just pulled this to check something for Translation and realized this field doesn't have an asterisk to mark it required. Can you add one? The `*` should be outside the `ts` like `<%= form.label :title, ts("Category name") + "*" %>`  so we don't have to worry about losing it in Translation.